pom.xml

```diff
@@ -83,6 +83,18 @@
             <timezone>Europe/Prague</timezone>
             <url>https://github.com/yruslan</url>
         </developer>
+        <developer>
```

Comment:
How come you two guys were missing, sorry.
Also I consider it a list of people who considerably, over longer time contributed to the codebase of the project.

Reply:
Thanks for adding us.